pywbem_mock/_mainprovider.py

```diff
@@ -678,12 +667,12 @@ def get_qualifier_store(self, namespace):
 
     def validate_namespace(self, namespace):
         """
-        Validates that a namespace is defined in the repository.
+        Validates that a namespace is defined in the datastore.
```

Comment:
I think the term "CIM repository" would be better than "datastore".

Reply:
Changed it to CIM repository rather than repository pretty much throughout the code and in messages.  Note. to be consistent we also changed the self.repository from comment above (which was self.datastore before) to self.cimrepository.

DONE